misc/config_tools/data/tgl-vecow-spc-7100-Corei7/tgl-vecow-spc-7100-Corei7.xml

```diff
@@ -0,0 +1,5092 @@
+	  50400000-506fffff : PCI Bus 0000:01
+	    50400000-504fffff : 0000:01:00.0
+	    50500000-505fffff : 0000:01:00.0
+	    50600000-50603fff : 0000:01:00.0
+	  50700000-5071ffff : 0000:00:1f.6
+	  50720000-50721fff : 0000:00:17.0
+	    50720000-50721fff : ahci
+	  50722000-507227ff : 0000:00:17.0
+	    50722000-507227ff : ahci
+	  50723000-507230ff : 0000:00:17.0
+	    50723000-507230ff : ahci
+	  50724000-50724fff : 0000:00:16.3
+	  50725000-50725fff : 0000:00:1f.5
+	c0000000-cfffffff : PCI MMCONFIG 0000 [bus 00-ff]
+	  c0000000-cfffffff : Reserved
+	fd000000-fd68ffff : pnp 00:0a
+	fd690000-fd69ffff : INT34C5:00
+	fd6a0000-fd6affff : INT34C5:00
+	fd6b0000-fd6cffff : pnp 00:0a
+	fd6d0000-fd6dffff : INT34C5:00
+	fd6e0000-fd6effff : INT34C5:00
+	fd6f0000-fdffffff : pnp 00:0a
+	fe000000-fe010fff : Reserved
+	fe04c000-fe04ffff : pnp 00:0a
+	fe050000-fe0affff : pnp 00:0a
+	fe0d0000-fe0fffff : pnp 00:0a
+	fe200000-fe7fffff : pnp 00:0a
+	fec00000-fec00fff : Reserved
+	  fec00000-fec003ff : IOAPIC 0
+	fed00000-fed00fff : Reserved
+	  fed00000-fed003ff : HPET 0
+	    fed00000-fed003ff : PNP0103:00
+	fed20000-fed7ffff : Reserved
+	  fed40000-fed44fff : MSFT0101:00
+	    fed40000-fed44fff : MSFT0101:00 MSFT0101:00
+	fed90000-fed90fff : dmar0
+	fed91000-fed91fff : dmar1
+	feda0000-feda0fff : pnp 00:09
+	feda1000-feda1fff : pnp 00:09
+	fedc0000-fedc7fff : pnp 00:09
+	fee00000-fee00fff : Local APIC
+	  fee00000-fee00fff : Reserved
+	ff000000-ffffffff : Reserved
+	  ff000000-ffffffff : pnp 00:0a
+	100000000-8afbfffff : System RAM
+	  408600000-409400ed0 : Kernel code
+	  409400ed1-409e535ff : Kernel data
+	  40a11d000-40a5fffff : Kernel bss
+	8afc00000-8afffffff : RAM buffer
+	4000000000-7fffffffff : PCI Bus 0000:00
+	  4000000000-400fffffff : 0000:00:02.0
+	    4000000000-40007e8fff : efifb
+	  4010000000-4016ffffff : 0000:00:02.0
+	  4017000000-4017000fff : 0000:00:10.0
+	    4017000000-40170001ff : lpss_dev
+	      4017000000-40170001ff : i2c_designware.0 lpss_dev
+	    4017000200-40170002ff : lpss_priv
+	  4017001000-4017001fff : 0000:00:15.0
+	    4017001000-40170011ff : lpss_dev
+	      4017001000-40170011ff : i2c_designware.1 lpss_dev
+	    4017001200-40170012ff : lpss_priv
+	    4017001800-4017001fff : idma64.1
+	      4017001800-4017001fff : idma64.1 idma64.1
+	  4020000000-40ffffffff : 0000:00:02.0
+	  6000000000-6000ffffff : 0000:00:02.0
+	  6001000000-60010fffff : 0000:00:1f.3
+	    6001000000-60010fffff : ICH HD audio
+	  6001100000-600110ffff : 0000:00:14.0
+	    6001100000-600110ffff : xhci-hcd
+	  6001110000-600111ffff : 0000:00:0d.0
+	    6001110000-600111ffff : xhci-hcd
+	  6001120000-6001123fff : 0000:00:1f.3
+	    6001120000-6001123fff : ICH HD audio
+	  6001124000-6001127fff : 0000:00:14.2
+	  6001128000-60011280ff : 0000:00:1f.4
+	  6001129000-6001129fff : 0000:00:16.0
+	    6001129000-6001129fff : mei_me
+	  600112b000-600112bfff : 0000:00:14.2
+	</IOMEM_INFO>
+  <BLOCK_DEVICE_INFO>
+	/dev/sda3: TYPE="ext4"
```

Comment:
My Vecow system has an NVMe SSD drive so shows up as
```
/dev/nvme0n1p2: TYPE="ext4" 
```
I think this difference will prevent me from using this board XML file for my system, right?

Reply:
Maybe, it is better to generate board.xml on your own board. Maybe we could consider to remove board.xml in future since it is better to generate board.xml on user's board.